assets/js/components/Login.jsx

```diff
@@ -64,7 +67,10 @@ export default function Login() {
                   name="username"
                   autoComplete="username"
                   required
-                  className="appearance-none block w-full px-3 py-2 border border-gray-300 rounded-md shadow-sm placeholder-gray-400 focus:outline-none focus:ring-jungle-green-500 focus:border-jungle-green-500 sm:text-sm"
+                  className={classNames(
+                    'appearance-none block w-full px-3 py-2 border border-gray-300 rounded-md shadow-sm placeholder-gray-400 focus:outline-none focus:ring-jungle-green-500 focus:border-jungle-green-500 sm:text-sm',
+                    { 'border-red-300': authError && authError.code === 401 }
```

Comment:
Could we wrap this in a function?
```
const unauthorized = (authError) => authError && authError.code === 401;
```

Reply:
Maybe just having a boolean `const` defined in the function body is enough, I was thinking about the function as well but I don't know if it's the best

Reply:
done